docs/docs/configuration/tooltip.md

```diff
@@ -60,14 +60,14 @@ Example:
 
 ```javascript
 /**
- * Custom positioner
- * @function Tooltip.positioners.custom
+ * External positioner
```

Comment:
I think `Custom` is better for positioners, because you can use those with the default tooltip.